source/SIL.AppBuilder.Portal.Frontend/src/data/containers/with-current-user.tsx

```diff
@@ -121,7 +121,13 @@ export function withCurrentUser(opts = {}) {
           const unauthorized = status === 401;
 
           if (status === 403 || status === 401) {
-            const errorJson = await tryParseJson(response);
+
+            let errorJson = {};
+            try {
+              errorJson = await tryParseJson(response);
+            } catch (e) {
+              // body is not json
```

Comment:
no console.warning? just swallow the error?

Reply:
good point.